pkg/signup/service/signup_service_test.go

```diff
@@ -198,8 +198,7 @@ func (s *TestSignupServiceSuite) TestSignupFailsWhenClientReturnsError() {
 
 	// when
 	_, err = s.Application.SignupService().Signup(ctx)
-	require.Error(s.T(), err)
-	require.Equal(s.T(), "an internal error:an internal error happened", err.Error())
+	require.EqualError(s.T(), err, "an internal error: an internal error happened", err.Error())
```

Comment:
`require.Error(s.T(), err)` was there to handle when `err` was `nil`?

Reply:
good point - when the error is nil, then it would panic in this case. It should have been just:
```suggestion
	require.EqualError(s.T(), err, "an internal error: an internal error happened")
```

Reply:
ah right, I made those changes using a search/replace but I missed the last `err.Error()` argument in some case. I'll fix that

Reply:
done in 90d0d39